proto/stride/stakeibc/query.proto

```diff
@@ -65,7 +65,11 @@ service Query {
     option (google.api.http).get = "/Stride-Labs/stride/stakeibc/epoch_tracker";
   }
 
-  // this line is used by starport scaffolding # 2
+  // Queries the next packet sequence for one for a given channel
+  rpc NextPacketSequence(QueryGetNextPacketSequenceRequest)
+      returns (QueryGetNextPacketSequenceResponse) {
+    option (google.api.http).get = "/Stride-Labs/stride/stakeibc/next_packet_sequence/{channel_id}/{port_id}";
```

Comment:
do you think we should add this query to `stakeibc` or `icacallbacks`?

Reply:
good question, I was wondering this as well. Maybe icacallbacks does make more sense? What do you think?

Reply:
I think icacallbacks might be slightly misleading cause this applies to all packets, not just ica's. But from a usage perspective, I think it aligns more with callbacks

Reply:
Stakeibc is just kind of a collect-all of misc stuff haha

Reply:
Chatted offline. It does seem more closely aligned with callbacks, but:
  * The purpose of the query is for stakeibc monitoring
  * Icacallbacks is treated more as public infra and if another project imported it, they would not expect that query (i.e. icacallbacks is more well scoped)

So the final consensus is to leave it in stakeibc for now